include/gdsb/experiment.h

```diff
@@ -68,6 +68,21 @@ template <typename It> void out(std::string name, It begin, It end, std::ostream
     stream << "]" << std::endl;
 }
 
+//! Use this function to output ranges in yaml format.
+template <typename It, typename F>
+void out(std::string name, It begin, It end, F const&& access, std::ostream& stream = std::cout)
+{
+    stream << name << ": [";
+
+    if (begin != end)
+    {
+        std::for_each(begin, end - 1, [&](auto const& e) { stream << access(e) << ", "; });
```

Comment:
Is it necessary to capture `&`? Why not capture `access` as `access = std::move(acces)`?